src/components/victory-area/victory-area.js

```diff
@@ -23,6 +23,42 @@ export default class VictoryArea extends React.Component {
   static role = "area";
 
   static defaultTransitions = {
+    onLoad: {
```

Comment:
fixed! https://github.com/FormidableLabs/victory-chart/pull/339/commits/b3ce2c4d7d7cedc30e9190cc15b9ab10aeab4b3e and https://github.com/FormidableLabs/victory-core/pull/108/commits/1d87a8144c3647aff556c3a60c09885da2e5a3ab